### PR TITLE
CRUD endpoints for prescriptions

### PR DESCRIPTION
**Pull request by rajwanialiya**

### CODE REVIEW

File: pages/api/prescriptions/update.ts [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  req: NextApiRequest,
+  res: NextApiResponse
+) {
+  if (req.method === 'POST') {
+    try {
+      const { id, name, status, pickupTime, balance, locationId, lockerBoxId } =
+        req.body
+
+      let prescription = null
+      prescription = await prisma.prescription.update({
```
Comment (JJwilkin) [inline pages/api/prescriptions/update.ts:15]: since we're not reassigning prescription it can just be `const prescription = await prisma...`

**Review by SammyRobensParadise — APPROVED** [review]
LGTM

**Review by hnagri52 — CHANGES_REQUESTED** [review]
missing migration file

File: prisma/schema.prisma [hunk 2]
```diff
@@ -54,8 +55,8 @@ model Prescription {
   Patient     Patient?  @relation(fields: [patientId], references: [id])
   patientId   Int
   balance     Float
-  location    Location  @relation(fields: [locationId], references: [id])
-  lockerBox   LockerBox @relation(fields: [lockerBoxId], references: [id])
```
Comment (hnagri52) [inline prisma/schema.prisma:58]: I might be wrong but I think prisma docs suggest model fields are kept lowercased

File: pages/api/prescriptions/read.ts [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+import prisma from '../../../lib/prisma'
+
+export default async function handler(
+  req: NextApiRequest,
+  res: NextApiResponse
+) {
+  // retrieve prescriptions by id, patientId, or don't specify body to retrieve all
+  if (req.method === 'POST') {
+    try {
+      const { id, patientId } = req.body
```
Comment (hnagri52) [inline pages/api/prescriptions/read.ts:12]: nit: will this throw if there is no id to be destructed?  I think we should separate our routes to follow general software practices:
```
/id
/all
```

Comment (rajwanialiya) [reply pages/api/prescriptions/read.ts:12]: since the else is a findMany, if theres no id specified all prescriptions are retrieved

**Review by JJwilkin — APPROVED** [review]

**Review by hnagri52 — APPROVED** [review]

**Pull request merged**